source/usr/lib/mcvirt/system.py

```diff
@@ -52,3 +52,14 @@ def getUserInput(display_text, password=False):
         else:
             sys.stdout.write(display_text)
             return sys.stdin.readline()
+
+    @staticmethod
+    def getNewPassword():
+        """Prompts the user for a new password, throwing an exception is the password is not
```

Comment:
sorry to be a pain, pylint will require the comment to be:

    def blah():
        """My multline
        comment
        """